.github/workflows/pre-release.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
What's the affect of this change?

Reply:
Only updating the semantic release dependency, they released a new version.